### PR TITLE
create a new file and add a console log #5

### PR DESCRIPTION
**Pull request by rohan-sorkar**
added a new file and wrote some lines of code that demonstrate the user's dream. it was implemented for issue #5. and I have done this by using a javascript file.

### CODE REVIEW

File: aboutMyDream.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('my dream is to be a intelligent programmer');
```
Comment (mohammadsaju) [inline aboutMyDream.js:1]: can you merge this "in near future" with that console log

Comment (rohan-sorkar) [reply aboutMyDream.js:1]: sure i can

**Comment by rohan-sorkar** [comment]
hey @mohammadsaju i have changed the code now you can review

**Review by mohammadsaju — APPROVED** [review]
well done my boy☺

**Pull request merged**